definitions/types/azure-security-iam.json

```diff
@@ -1,24 +1,34 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema",
   "type": "object",
-  "title": "Azure Security IAM Information",
-  "description": "Informs downstream services of IAM roles required for operation",
+  "title": "IAM",
+  "description": "IAM Roles And Conditions",
   "additionalProperties": false,
-  "required": [
-    "roles",
-    "scope"
-  ],
-  "properties": {
-    "roles": {
-      "title": "List of Azure IAM roles",
-      "type": "array",
-      "items": {
-        "type": "string"
+  "patternProperties": {
+    "^[a-z]+$": {
+      "type": "object",
+      "required": [
+        "role",
+        "scope"
+      ],
+      "properties": {
+        "role": {
+          "title": "Role",
```

Comment:
Make this a string (single) instead of an array of strings since the policies block in applications is now an array.

Reply:
Sounds good